test/blocks/text/text.test.js

```diff
@@ -62,4 +62,11 @@ describe('text block', () => {
       });
     });
   });
+
+  describe('per-viewport content', () => {
+    it('two content rows has viewport classes', () => {
```

Comment:
nit: `it` clauses should start with verbs so it would read `per-viewport content has viewport classes if there are two content rows` or change the describe so it is `two content rows` and it is `has viewport classes`.